src/mscorlib/corefx/System/Globalization/TextInfo.cs

```diff
@@ -38,38 +36,28 @@ public partial class TextInfo : ICloneable, IDeserializationCallback
         ////
 
         [OptionalField(VersionAdded = 2)]
-        private String m_listSeparator;
```

Comment:
> they expect

To be clear, the "they" here that opened that particular bug are the implementers of that serialization mechanism.

> it will be strange to support DCS on cross-platforms and not supporting binary serialization

I don't see why.  They're based on two different approaches to serialization.

Would it be a nice feature?  Of course.  If we get the latter for free with the former, great.  If in the fullness of time we can make the latter work regardless, great.  But I don't believe it's possible or desirable to make that a goal now.  To make that completely work, you will need to revisit the implementations of hundreds upon hundreds of types across coreclr and corefx, adding new ISerializable/etc. implementations to many of them, and establishing test infrastructure that's able to verify the same serialization/deserialization of all serialiable types across Windows/OSX/Linux, 64-bit/32-bit, desktop/coreclr/corert, etc.

It's of course possible I'm overestimating the cost here.  I don't believe I am, though.